cdap-ui/app/hydrator/controllers/create/toppanel-ctrl.js

```diff
@@ -294,10 +304,36 @@ class HydratorPlusPlusTopPanelCtrl {
     seconds = seconds < 10 ? '0' + seconds : seconds;
     minutes = minutes < 10 ? '0' + minutes : minutes;
 
+    this.setDisplayDuration(minutes, seconds);
+  }
+
+  setDisplayDuration(minutes, seconds) {
     this.displayDuration = {
-      minutes: minutes,
-      seconds: seconds
-    };
+      minutes: !minutes ? '--' : minutes,
```

Comment:
Would this be easier to read as `minutes ? minutes : '--'` or even `minutes || '--'`? It looks like we display `--` for `0`.

Reply:
Done